pageserver/src/http/openapi_spec.yml

```diff
@@ -251,6 +251,13 @@ paths:
             type: string
             format: date-time
           description: A timestamp to get the LSN
+        - name: with_lease
```

Comment:
The mirrored spec in `neondatabase/cloud` will have to be updated, but that can happen when the control plane team does the enablement work on their side.

Reply:
Will address in a followup.